web/themes/custom/usagov/templates/html.html.twig

```diff
@@ -81,10 +81,8 @@
   <meta name="description" content="¿Ha sufrido una estafa y quiere reportarla? Solo responda unas cuantas preguntas para saber cómo reportar la estafa en el lugar correcto." />
   {% endif %}
   <!-- Favicons -->
-  <link rel="apple-touch-icon-precomposed" href="/themes/custom/usagov/images/USA_Fav_Icon152_default.png">
-  <link rel="icon" sizes="16x16" href="/themes/custom/usagov/images/USA_Fav_Icon16.ico">
-  <link rel="icon" sizes="32x32" href="/themes/custom/usagov/images/favicon.ico">
-  <link rel="icon" sizes="152x152" href="/themes/custom/usagov/images/USA_Fav_Icon152_default.png">
+  <link rel="icon" href="web/themes/custom/usagov/images/Favicon_USAGov.png">
```

Comment:
These were both 404 for me. It looks like they should be in /themes, not web/themes.